packages/office-ui-fabric-react/src/components/DatePicker/DatePicker.base.tsx

```diff
@@ -220,10 +220,22 @@ export class DatePickerBase extends BaseComponent<IDatePickerProps, IDatePickerS
     }
   }
 
-  private _renderTimePicker() {
+  public calculatingTime(newtime: string, newDate?: Date) {
+    const time = this._parseHourAndTime(newtime);
+    time.hour = (time.hour) ? time.hour : 0;
+    time.minute = (time.minute) ? time.minute : 0;
+
+    // Return the correct date object with the time modified
+    const updatedDate = (newDate ? newDate : this.state.selectedDate as Date);
+    updatedDate.setHours(time.hour, time.minute);
 
+    return updatedDate;
   }
 
+  // private _renderTimePicker() {
+
+  // }
+
```

Comment:
Is this needed

Reply:
No, (if I remember correctly) the errors were that private functions had to be placed after all public functions and empty functions weren't allowed, I left it commented in case it was a placeholder but we can remove it

Reply:
Let's remove it then